framework/src/modules/chain/dpos/delegates_info.js

```diff
@@ -76,7 +76,11 @@ class DelegatesInfo {
 	 * @param {Block} block
 	 */
 	async _update(block, undo, tx) {
-		// @todo add proper description
+		/**
+		 * If the block height is 1, that means the block is
+		 * the genesis block, in that case we don't have to
+		 * update anything in the accounts.
+		 */
 		if (block.height === 1) {
 			return false;
```

Comment:
what does returning false/true signify here? Also, out of scope, but don't we need to update the `_updateVotedDelegatesVoteWeight` for geneisBlock?

Reply:
Nothing actually. It's just the termination of the method. 

Based on the previous rounds implementation it seems we were not doing it. But if there is a case that genesisBlock can contain vote transactions from a delegate to someone else maybe we should do it? @shuse2 ?

Reply:
GenesisBlock contains delegate registeration as well as vote transactions. I wonder how it was syncing with the testnet. @lsilvs what do you think?

Reply:
dpos module only do the apply for `reward` and `fee`, which does not exists on the genesis block now.
Therefore, in our case it should work now.

Reply:
Among all updates performed during `Dpos.apply`, only `_updateProducedBlocks` would be relevant for genesis block because genesis block is not the last block of the round. As the genesis block forger is not even a delegate account, we don't update its `producedBlocks` column.

PS: Note that `_updateVotedDelegatesVoteWeight` is not related to vote transactions included in the block. It's related to the accounts that the block forger voted for.